DeepCat.js

```diff
@@ -473,8 +473,13 @@
 		return false;
 	}
 
+	function setImeAndSuggestions( enable ) {
+		$( '.suggestions' ).css( 'z-index', enable === false ? -1 : 'auto' );
+		$( '.imeselector' ).css( 'z-index', enable === false ? -1 : 'auto' );
+	}
+
 	/**
-	 * Hash function for generating hint box cookie token.
+	 * Hash function for generating hint box setting token.
```

Comment:
"hint box setting" sounds really mysterious if you don't know exactly what this is about
How about "hide hint (box)" instead of "hint box" all over the code?